docs/setup_guides/hpc.rst

```diff
@@ -117,6 +120,14 @@ securing Jupyter, check out
 `Securing a notebook server <http://jupyter-notebook.readthedocs.io/en/stable/public_server.html#securing-a-notebook-server>`__
 in the Jupyter documentation.
 
+Finally, we may want to configure dask's dashboard to forward through jupyterhub.
+This can be done by editing the dask distributed config file, e.g.:
+``.config/dask/distributed.yaml``. In this file, set:
+
+.. code:: python
+
+    diagnostics-link: "../proxy/{port}/status"
```

Comment:
Classic Jupyter notebooks and JupyterLab have different relative paths.  I wonder if a static path like `/proxy/{port}/status` can be made to work.